### PR TITLE
113797 add exclamation triangle icon

### PR DESCRIPTION
**Pull request by hyperd**
### Review
 - [x] @boskya 

### After Merge

run:

```
git checkout master
git pull
npm version patch
npm publish
```

### note

If npm version is `2.13` or later npm version will likely fail to run postversion script, leaving you with pending commits in your repo.

If this is the case, run `npm run-script postversion` to finish the process.

### CODE REVIEW

**Comment by boskya** [comment]
@hyperd can you use pullquester for this PR. It has instructions for publishing. if you don't have have pullquest, npm install -g pullquester and the way you run it is pull.

**Comment by hyperd** [comment]
Ok. Forgot that was in the readme.

File: .pullquester/pullrequest.json [hunk 1]
```diff
@@ -19,10 +19,12 @@
...
             "name": "Joel Brighton",
             "value": "@joebri"
-        },
+        }
+    ],
+    "testers": [
         {
-            "name": "Jonathan Park",
-            "value": "@park9140"
+            "name": "Phillip Balsley",
```
Comment (park9140) [inline .pullquester/pullrequest.json]: Did you test this @pbalsley?

File: styles/iconography/_styles.scss [hunk 2]
```diff
@@ -47,6 +47,10 @@
         @include icon-small-block($fa-var-user);
     }
 
+    &.exclamation-triangle {
+        @include icon-small(before, right, $fa-var-exclamation-triangle);
```
Comment (park9140) [inline styles/iconography/_styles.scss]: This looks correct, although you should really probably be using a second class to change the size.  `exclamation-triangle small-icon`

Comment (boskya) [inline styles/iconography/_styles.scss]: or .icon .exclamation-triage .small. But we are not following that pattern consistently...looks fine for now.

File: .pullquester/pullrequest.json (hunk 1 above)
Comment (boskya) [inline .pullquester/pullrequest.json]: FYI @hyperd @pbalsley  does not sign off on style guide changes. So no need to add him here.

**Comment by boskya** [comment]
remove @pbalsley from the pull quester config and then :shipit:

**Pull request merged**